hushline/model/field_definition.py

```diff
@@ -20,16 +20,20 @@ class FieldDefinition(Model):
     __tablename__ = "field_definitions"
     __table_args__ = (UniqueConstraint("username_id", "sort_order"),)
 
-    id: Mapped[int] = mapped_column(primary_key=True)
-    username_id: Mapped[int] = mapped_column(db.ForeignKey("usernames.id"), index=True)
+    id: Mapped[int] = mapped_column(primary_key=True, nullable=False)
```

Comment:
all the int/PK columns can have auto_increment added which may help silence some warnings? I think this is the preferred thing to do anyway for clarity.